test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/ExpressionMetadataProviderTest.cs

```diff
@@ -159,5 +225,11 @@ private class Category
 
             public string CategoryName { get; set; }
         }
+
+        private string PrivateProperty { get; set; }
+
+        public static string StaticProperty { get; set; }
+
+        public string Field = "Hello";
```

Comment:
Probably better to place these at the top of the class instead of below the nested classes.